Code of conduct in French

LoicB opened the pull request:
## What does this PR do?
Add resource(s) code of conduct in French

## For resources
### Description
Translation of the Code of conduct into French.

### Why is this valuable (or not)?

It will help French-speaking readers.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

davorpa reviewed (changes requested):
Incomplete.

Doesn't fit with original: https://www.contributor-covenant.org/fr/version/1/3/0/code-of-conduct/code_of_conduct.md

eshellman commented:
please respond to @davorpa I'm confused

LoicB commented:
Hi @davorpa, thank you for your feedback.
I have updated the code of conduct.
I used mainly the original and modified what I thought was necessary.
Is what you meant?
Thanks

davorpa commented on CODE_OF_CONDUCT-fr.md:
```diff
@@ -0,0 +1,53 @@
+circonstances. Les mainteneurs s'obligent à garder confidentielles les
+informations de la personne qui remonte un incident.
+
+Ce Code de Conduite est adaptée du [Contributor Covenant](https://.contributor-covenant.org),
+version 1.3.0, disponible à https://.contributor-covenant.org/fr/version/1/3/0/code-of-conduct.html
+
+[page d'accueil]: https://contributor-covenant.org
```
- Fix links to be rendered :wink:
- Use right markdown syntax to link as foo notes

```suggestion
Ce Code de Conduite est adaptée du [Contributor Covenant](homepage),
version 1.3.0, disponible à https://contributor-covenant.org/fr/version/1/3/0/code-of-conduct.html

[homepage]: https://www.contributor-covenant.org
```
